tiledb/multirange_indexing.py

```diff
@@ -287,6 +287,31 @@ def __getitem__(self, idx):
         if self.use_arrow:
             import pyarrow as pa
 
+            # TODO currently there is lack of support for Arrow list types. this
```

Comment:
Please move this check to a new conditional block above the `results = super...` section so that we error out before  submitting the query.